# Description
- Title: **Gut microbiome study of inflammatory bowel disease in a Korean population cohort**
- Authors: **Hyun Sik Kim, Jae-Yun Lee, Bo-Hyung Kim, and Chang Kyun Lee**
- Year: **2025**

# Contents
**- `Data`: A directory containing processed data in analysis-ready format.**
  - `ASV_table.csv` [comma-separeted table]: Count table of amplicon sequencing variants (ASVs) across the samples. Rows are ASVs and columns are samples. This table was generated by [DADA2](https://benjjneb.github.io/dada2/index.html) pipeline through [Qiime2](https://qiime2.org/) platform.
  - `ASV_rep-seqs.fastq` [FASTA]: Sequences of each ASV, which used for contruction of phylogenetic tree or taxonomic assignment of ASVs. This FASTA file was generated from [DADA2](https://benjjneb.github.io/dada2/index.html) pipeline through [Qiime2](https://qiime2.org/) platform.
  - `Phylogenetic_tree.nwk`[Newick]: Phylogenetic tree of ASVs constructed by [q2-fragment-insertion](https://library.qiime2.org/plugins/q2-fragment-insertion/16/) plugin in [Qiime2](https://qiime2.org/) platform.
  - `Taxonomy.csv` [comma-separated table]: Taxonomy assignment of ASVs. Generated from [Qiime2](https://qiime2.org/) platform and [SILVA](https://www.arb-silva.de/) database.
  - `Metadata.csv` [comma-separated table]: Metadata containing six columns
    - _SampleID_ [character]: The unique identifier for each of the 640 samples collected from the study participants.
    - _Group_ [categorical]: Indicates the clinical group from which the sample was obtained. This column consists of three categories: ‘CD’, ‘UC’, and ‘Healthy’.
    - _Age_ [integer]: The age of the participant at the time of sample collection.
    - _Gender_ [categorical]: The biological sex of the participant from whom the sample was obtained. The values include ‘Male’ and ‘Female’.
    - _Disease behavior(CD)_ [categorical]: This column contains information on the disease behavior of the patients with CD. It reflects the current severity and pattern of CD, classified as ‘inflammatory’, ‘stricturing’, or ‘penetrating’. For samples from non-CD groups, the value is assigned as ‘None’.
    - _Disease location(UC)_ [categorical]: This column records the disease extent in the patients with UC. It describes how extensively the disease has spread. Samples from non-UC groups are assigned the value ‘None’. For UC group, the values include ‘proctitis’, ‘left_sided’, and ‘extensive’.


- `Analysis_codes`
  - `Primer_trimming.py`: Python script used for trimming 338F and 805R primer from the raw reads using [Cutadapt](https://cutadapt.readthedocs.io/en/stable/) before DADA2 pipeline.
    ```bash
    # Example usage
    chmod 755 Primer_trimming.py # Set permissions for the script
    ./Primer_trimming.py -h # For more detail
    ```
  - `Draw_figures.R`: R script used for generating the figures presented in our paper. All the data used for this script are in `Processed_data.zip`.
